waspc/test/SemanticVersion/VersionBoundTest.hs

```diff
@@ -0,0 +1,65 @@
+module SemanticVersion.VersionBoundTest where
+
+import Test.Tasty.Hspec
+import Wasp.SemanticVersion.Version
+import Wasp.SemanticVersion.VersionBound hiding ((∈), (∩), (∪), (⊆))
```

Comment:
Ah ok party poopers :(. No it is not, I added it because they were perfect fit and in my opinion made code more readable: but do note I added them as alternative / aliases, you can use functions with normal names instead if you want (and we don't use unicodes anyqwere really).
I removed them now.